### PR TITLE
Added pdf resources for html study

### PR DESCRIPTION
**Pull request by noobkid2411**
PDF material for introductory course on web developement. 
Kindly check the request and please add hacktober-accepted if the resources seem valuable.Thanks!

### CODE REVIEW

**Review by blank-27 — APPROVED** [review]
looks good to me

**Comment by 3scava1i3r** [comment]
please join the [discord](https://discord.gg/6QAjyBa) server here and add your discord handle in the PR

**Review by aksh1407 — CHANGES_REQUESTED** [review]
We would prefer links rather than pdfs ripped off of web pages. Kindly update the readme files in respective folders with useful resource links rather than the pdf files. Thank You.

**Comment by noobkid2411** [comment]
> We would prefer links rather than pdfs ripped off of web pages. Kindly update the readme files in respective folders with useful resource links rather than the pdf files. Thank You.

Edited and added resources' link to the resources . Kindly check now.

**Review by aksh1407 — CHANGES_REQUESTED** [review]
Kindly remove files from Designing directory as well.

**Comment by blank-27** [comment]
@noobkid2411 please stop spamming here. Join over discord server if you need some help ,we are happy to help

**Comment by noobkid2411** [comment]
Yes,sorry for leaving out.Done now,hope it's alright now!

**Review by blank-27 — CHANGES_REQUESTED** [review]
need changes

**Comment by archana-17** [comment]
Please check in the ML resources "Curated list of AI/ML resources" link is not working. @noobkid2411

File: Web Development/Resources/README.md [hunk 1]
```diff
@@ -3,7 +3,11 @@
 - [The Complete 2020 Web Development Bootcamp](https://www.udemy.com/course/the-complete-web-development-bootcamp/)
 - [The Web Developer Bootcamp](https://www.udemy.com/course/the-web-developer-bootcamp/)
 - [10 Web Development YouTube Channels You Probably Didn't Know About](https://dev.to/ryandsouza13/10-web-development-youtube-channels-you-probably-didn-t-know-about-4o37?utm_source=digest_mailer&utm_medium=email&utm_campaign=digest_email)
+- [Web developement courses on coursera](https://www.coursera.org/courses%3Fquery%3Dweb%2520development&ved=2ahUKEwiLzJXwlKLsAhV8wTgGHfi6AG8QFjAJegQIIBAH&usg=AOvVaw3RaH7LlNPk66swGThS5zNv)
```
Comment (aksh1407) [inline Web Development/Resources/README.md:6]: This link does not work. Even if it did, why would someone use this link to search Web Development Courses on Coursera when they can do that already on Coursera itself.

File: Web Development/Resources/README.md [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+- [Best web developement courses for beginners](https://studywebdevelopment.com/web-development-courses.html)
```
Comment (aksh1407) [inline Web Development/Resources/README.md:7]: Same is the case with this.

File: Web Development/Resources/README.md [hunk 3]
```diff
@@ -3,7 +3,11 @@
...
+- [Complete web developement course by edureka](https://m.youtube.com/watch%3Fv%3DQ33KBiDriJY&ved=2ahUKEwiLzJXwlKLsAhV8wTgGHfi6AG8QwqsBMA56BAgSEAM&usg=AOvVaw1-yMWJLtJSaRL4WFL1vAiC)
```
Comment (aksh1407) [inline Web Development/Resources/README.md:8]: This is a link for Smartphone browsers.

**Review by aksh1407 — COMMENTED** [review]
These links are not of much use.

**Comment by aksh1407** [comment]
I am closing this PR since it is cluttered and does not contain content of much use. Create a fresh PR with helpful and usable content for further contribution.

**Pull request closed without merging**